src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -1159,6 +1169,34 @@ class AddItemsPage extends Component {
 
     return Promise.resolve();
   }
+  saveRequisitionItemsInCurrentStepWithAlert({
+    lineItems,
+    beforeCB = () => {},
+    thenCB = () => {},
+    catchCB = () => {},
+  }) {
+    confirmAlert({
+      title: this.props.translate('react.stockMovement.message.confirmSave.label', 'Confirm save'),
+      message: this.props.translate(
+        'react.stockMovement.QOHWillNotBeSaved.message',
+        'This save action won’t save the quantity on hand you have entered. You will have to reenter these when you came back to this request later. Are you sure you want to proceed?',
+      ),
+      buttons: [
+        {
+          label: this.props.translate('react.default.yes.label', 'Yes'),
+          onClick: () => {
+            beforeCB();
```

Comment:
Just show spinner, what's the point of doing this `beforeCB`

Reply:
the CB stands for callback but I have already changed this property as @awalkowiak  requested